ansible/roles/loadbalancer/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+# This role will install Loadbalancer in group 'loadbalancers' in the environment inventory
+
+- name: pull the lastest image of controller
```

Comment:
@lehoanganh change to "loadbalancer". also typo "lastest". also "latest" would be wrong. it is not the latest image that we pull but the one that is passed as tag. I would change it to "required image".